File: backend/chat-service/src/main/java/com/example/chatservice/chat/service/ChatRoomMemberService.java
package com.example.chatservice.chat.service;

import com.example.chatservice.chat.repository.ChatRoomMemberRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

@Service
@Transactional
@RequiredArgsConstructor
public class ChatRoomMemberService {

    private final ChatRoomMemberRepository chatRoomMemberRepository;
}

File: backend/chat-service/src/main/kotlin/com/example/chatservice/chat/service/ChatRoomMemberService.kt
package com.example.chatservice.chat.service

import com.example.chatservice.chat.repository.ChatRoomMemberRepository
import org.springframework.stereotype.Service
import org.springframework.transaction.annotation.Transactional

@Service
@Transactional
class ChatRoomMemberService(private val chatRoomMemberRepository: ChatRoomMemberRepository)
